gdal/swig/include/cpl.i

```diff
@@ -334,6 +334,104 @@ char **wrapper_VSIReadDirEx( const char * utf8_path, int nMaxFiles = 0 )
 char **VSIReadDirRecursive( const char * utf8_path );
 %clear char **;
 
+#ifdef SWIGPYTHON
+%rename (OpenDir) wrapper_VSIOpenDir;
+%inline {
+VSIDIR* wrapper_VSIOpenDir( const char * utf8_path,
+                            int nRecurseDepth = -1,
+                            char** options = NULL )
+{
+    return VSIOpenDir(utf8_path, nRecurseDepth, options);
+}
+}
+
+%{
+typedef struct
+{
+    char*        name;
+    int          mode;
+    GIntBig      size;
```

Comment:
this is a vsi_l_offset in the CPL definition...

Reply:
yeah, there is no vsi_l_offset typemap in SWIG. For VSIStatL() we use GIntBig as well. This has no practical impact